docs/pages/management/admin/adding-nodes.mdx

```diff
@@ -65,7 +65,7 @@ Auth Service. To receive a host certificate upon joining a cluster, a new
 Teleport host must present an **invite token**.
 
 An invite token also defines which role a new host can assume within a cluster
-(e.g., `auth`, `proxy`, `node`, or `kube`).
+(`proxy`, `node`, `auth`, `app`, `db`, `kube`, or `windowsdesktop`).
```

Comment:
If we're going to use this paragraph to list all possible roles, rather than a few example ones, I think an unordered list would look better than parentheses.

Reply:
What does an unordered list look like in this context? Do you mean a bullet point list like

```
- proxy
- node
- etc
```

Reply:
That's right

Reply:
520ad57708f0af035518e1f0586f35add6dafee7